core/chains/evm/client/errors_test.go

```diff
@@ -51,6 +51,23 @@ func Test_Eth_Errors(t *testing.T) {
 		}
 	})
 
+	t.Run("IsTransactionAlreadyMined", func(t *testing.T) {
+		assert.False(t, randomError.IsNonceTooLowError())
```

Comment:
This says IsNonceTooLowError, was that intentional?

Reply:
I thought I had properly removed that... Pushing the removal change in a moment.